### PR TITLE
Updates to terra-clinical to fix the Heroku slug size of the project

### PR DESCRIPTION
**Pull request by ryanthemanuel**
### Summary
This PR will fix issue #77  It moves terra-toolkit to be defined as a dependency at the top level only.

### CODE REVIEW

File: packages/terra-clinical-action-header/tests/nightwatch.conf.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 /* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```
Comment (emilyrohrbough) [inline packages/terra-clinical-action-header/tests/nightwatch.conf.js]: Can we remove this line?

File: packages/terra-clinical-action-header/package.json [hunk 2]
```diff
@@ -21,9 +21,6 @@
     "url": "https://github.com/cerner/terra-clinical/issues"
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
-  "devDependencies": {
-    "terra-toolkit": "^0.x"
-  },
```
Comment (Matt-Butler) [inline packages/terra-clinical-action-header/package.json]: Do we want to do this in terra-core as well? And update the generator?

Comment (ryanthemanuel) [inline packages/terra-clinical-action-header/package.json]: We should definitely update the generator.  We can't do it in terra-core.  It's a chicken and an egg kind of thing unfortunately.  Can't depend on something at the root level if it's a package

**Comment by Matt-Butler** [comment]
I am curious, we have `nightwatch.conf.js` the same in each package. Can we instead just reference a `nightwatch.conf.js` in the root? I don't want this to hold up the review, maybe we could just log it as an issue.

**Comment by emilyrohrbough** [comment]
@Matt-Butler So I have been wondering about that. I actually think we have some dead code in both core and clinical. From what I can tell, the `nightwatch.conf.js` file is never referenced from a package level, nor from the root. I think it is only being referenced from the site package. It might be something we want to check out.

File: packages/terra-clinical-action-header/tests/nightwatch.conf.js (hunk 1 above)
Comment (ryanthemanuel) [inline packages/terra-clinical-action-header/tests/nightwatch.conf.js]: Good call.  I'll remove the extraneous ones.

**Comment by emilyrohrbough** [comment]
Slug size is now 113M with this change.

File: packages/terra-clinical-action-header/tests/nightwatch.conf.js (hunk 1 above)
Comment (ryanthemanuel) [inline packages/terra-clinical-action-header/tests/nightwatch.conf.js]: Done

**Pull request merged**